webpack.conf.js

```diff
@@ -1,25 +1,46 @@
 var prebid = require('./package.json');
 var StringReplacePlugin = require('string-replace-webpack-plugin');
 var path = require('path');
+var webpack = require('webpack');
+var helpers = require('./gulpHelpers');
+var RequireEnsureWithoutJsonp = require('./plugins/RequireEnsureWithoutJsonp.js');
+
+// list of module names to never include in the common bundle chunk
+var neverBundle = [
+  'AnalyticsAdapter.js'
+];
 
 module.exports = {
-  output: {
-    filename: 'prebid.js'
-  },
   devtool: 'source-map',
   resolve: {
-    modulesDirectories: ['', 'node_modules', 'src']
+    root: [
+      path.resolve('.')
+    ],
+    modulesDirectories: ['', 'node_modules']
   },
   resolveLoader: {
-    modulesDirectories: ['loaders', 'node_modules']
+    root: [
+      path.resolve('./loaders'),
+      path.resolve('./node_modules')
+    ]
+  },
+  output: {
+    jsonpFunction: 'pbjsChunk'
   },
   module: {
     loaders: [
       {
         test: /\.js$/,
-        include: /(src|test)/,
-        exclude: path.resolve(__dirname, 'node_modules'),
-        loader: 'babel', // 'babel-loader' is also a legal name to reference
+        exclude: path.resolve('./node_modules'), // required to prevent loader from choking non-Prebid.js node_modules
+        loader: 'babel',
+        query: {
+          presets: ['es2015']
+        }
+      },
+      { // This makes sure babel-loader is ran on our intended Prebid.js modules that happen to be in node_modules
```

Comment:
Internal modules are stored in `modules` the directory.  This pull-requests also adds the ability for modules to come from `node_modules` i.e. modules that come from npm.  This is briefly discussed in the `External modules` sections I provided in the documentation above.